mmocr/datasets/preparers/parsers/loaders.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import re
+from typing import Dict, Tuple, Union
+
+import yaml
+
+
+def txt_loader(file_path: str,
+               separator: str = ',',
+               format: str = 'x1,y1,x2,y2,x3,y3,x4,y4,trans',
+               encoding='utf-8') -> Union[Dict, str]:
+    """Loading txt format annotations.
+
+    Args:
+        file_path (str): Path to the txt file.
+        separator (str, optional): Separator of data. Defaults to ','.
+        format (str, optional): Annotation format.
+            Defaults to 'x1,y1,x2,y2,x3,y3,x4,y4,trans'.
+        encoding (str, optional): Encoding format. Defaults to 'utf-8'.
+
+    Yields:
+        Iterator[Union[Dict, str]]: Original text line or a dict containing
+        the information of the text line.
+    """
+    keys = format.split(separator)
+    with open(file_path, 'r', encoding=encoding) as f:
+        for line in f.readlines():
+            line = line.strip()
+            if line:
+                yield dict(zip(keys, line.split(separator)))
+
+
+def totaltext_loader(file_path: str) -> str:
+    """The annotation of the totaltext dataset may be stored in multiple lines,
+    this loader is designed for this special case.
```

Comment:
If it's just a special case, shall we move this loader into `TotaltextTextDetParser` instead of spreading these utilities everywhere?